src/main/scala/com/stratio/hermes/helpers/ResourcesHelper.scala

```diff
@@ -0,0 +1,37 @@
+
+package com.stratio.hermes.helpers
+
+import java.io.{File, InputStream}
+
+import com.stratio.hermes.exceptions.HermesException
+import com.stratio.hermes.implicits.HermesSerializer
+import org.json4s.native.Serialization.read
+
+import scala.util.{Failure, Success, Try}
+
+object ResourcesHelper extends HermesSerializer {
+
+  def getResources(name: String): Seq[String] = Try(
+    new File(getClass.getResource(s"/locales/$name").getFile)) match {
+    case Success(resources) => resources.list().toSeq
+    case Failure(_) => throw new HermesException(s"Error loading invalid name /locales/$name")
+  }
+
+  def getResource(name: String, file: String): InputStream = {
+    Option(getClass.getResourceAsStream(s"/locales/$name/$file")) match {
+      case Some(resource) => resource
+      case _ => throw new HermesException(s"Error loading invalid resource /locales/$name/$file")
+    }
+  }
+
```

Comment:
I'd extract the following two methods to its own class or trait (KhermesParser??)